google/ggrc-core
# GGRC-7585 UI is broken for Description field

# Issue description

When rich-text field are initializing user sometimes see rich-text toolbar w/o styles.

# Steps to test the changes

Precondition:
1. Open chrome dev tools
2. Open rich-text.js file on source tab
3. Find initEditor() method
4. Put breakpoint on row with then() after quill import
It helps simulate delay when rich-text component was rendered but `quill` hasn't initialized
5. Open any object with description
6. Review rich-text field
7. Expected: It is empty

# Solution description

Made toolbar hidden when rich-text has not focus

# Sanity checklist




# PR Review checklist
